content/ux-strat-bootcamp/week-one/thursday/data-dashboard-analysis/_index.md

```diff
@@ -0,0 +1,25 @@
+---
+title: Data Dashboard Analysis
+content_type: project
+ready: true
+---
+
+A data dashboard is an information management tool that visually tracks, analyses and displays key performance indicators (KPI), metrics and key data points to monitor the health of a business, department or specific process.
+
+A wireframe is a simple visual guide that represents the skeletal framework of a digital product. 
+
+**Task**
```

Comment:
It would be better to use Markdown's formatting the way it is defined.

For example if you want to make a Title, you give it one hash. If you want to make a heading, use 2 hashes, a sub heading gets 3, a sub sub heading gets 4. Like this:

# Title 
## Heading 
### Sub heading
#### Sub sub heading

Reply:
I would suggest giving Task 2 # symbols.

Eg:

## Task